odh/base/monitoring/kustomization.yaml

```diff
@@ -4,3 +4,8 @@ kind: Kustomization
 namespace: opf-monitoring
 resources:
   - kfdef.yaml
+  - prometheus.yaml
+  - prometheus-route.yaml
+  - grafana.yaml
```

Comment:
these 3 are now created via the kfdefs, so they shouldn't be here (the prometheus/grafana instancess, and prometheus route)